shared/modules/bus_module.go

```diff
@@ -27,6 +25,6 @@ type Bus interface {
 	GetTelemetryModule() TelemetryModule
 
 	// Configuration
-	GetConfig() map[string]json.RawMessage
-	GetGenesis() map[string]json.RawMessage
+	GetConfig() Config
```

Comment:
Why are these methods re-introduced?

Reply:
Addressed in https://github.com/pokt-network/pocket/pull/235/commits/2837b5aff91973a67988c6d5db9ae893a6445437